apps/sms/lib/bridge/bridge.js

```diff
@@ -32,13 +32,27 @@ module.exports = Client;
 /**
```

Comment:
@azasypkin It didn't need to split the client/service in this patch. I just want to make sure that we only need client in window context after moving `ActivityShim` to iframe. Sorry about the confusion :/